### PR TITLE
Spanish Translation

### PR DESCRIPTION
**Pull request by CloverBadland**
I've added the spanish tanslation for strings and F-Droid metadata. I've added extra information to make the translation into Spanish more natural.

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <string name="assistant_title">Aplicación de Asistencia del Sistema</string>
+    <string name="assistant_content">Establece esta aplicación como la aplicación de asistencia del sistema. Úsalo manteniendo presionado el botón de inicio o deslizando hacia arriba desde el borde inferior.</string>
+    <string name="assistant_open_settings">Abrir configuraciones del asistente</string>
+
+    <string name="setting_title">Configuración</string>
+    <string name="setting_action_save">Guardar</string>
+    <string name="setting_message_saved">Guardado</string>
+
+    <string name="setting_keyword_title">Filtro de Palabras Clave</string>
+    <string name="setting_keyword_content">Si está habilitado, ClipboardBoard solamente vaciará el contenido que contenga estas palabras clave.</string>
```
Comment (CloverBadland) [inline app/src/main/res/values-es/strings.xml:63]: ClipboardBoard or ClipboardCleaner?

Comment (DeweyReed) [reply app/src/main/res/values-es/strings.xml:63]: ClipboardCleaner

Comment (DeweyReed) [reply app/src/main/res/values-es/strings.xml:63]: It's typo. I'll fix it later.

**Comment by DeweyReed** [comment]
Thank you! I'll release the new version in the next few days.

**Pull request merged**